### PR TITLE
AppLovin/remove_mrec_and_native_ads

### PR DESCRIPTION
**Pull request by tiandrew**

### CODE REVIEW

File: ThirdPartyAdapters/applovin/applovin/src/main/java/com/google/ads/mediation/applovin/AppLovinMediationAdapter.java [hunk 1]
```diff
@@ -80,21 +80,17 @@
   @IntDef(
       value = {
         ERROR_BANNER_SIZE_MISMATCH,
-        ERROR_REQUIRES_UNIFIED_NATIVE_ADS,
```
Comment (joshuajplagonera) [inline ThirdPartyAdapters/applovin/applovin/src/main/java/com/google/ads/mediation/applovin/AppLovinMediationAdapter.java:83]: Please revert the changes to this file. The error codes should not be changed, even though they are unused.

Comment (tiandrew) [reply ThirdPartyAdapters/applovin/applovin/src/main/java/com/google/ads/mediation/applovin/AppLovinMediationAdapter.java]: Error code changes have been reverted

**Comment by joshuajplagonera** [comment]
Obsolete PR. Closing...
